ci/scripts/pre-check.sh

```diff
@@ -5,11 +5,13 @@ set -euo pipefail
 
 source ci/scripts/common.env.sh
 
+echo "--- Check release mod"
+cargo check --release --features "static-link static-log-level"
```

Comment:
Let's still run `cargo clippy` here like the following 2 lines. `cargo clippy` contains all the content of `cargo check`.